README.md

```diff
@@ -75,12 +77,12 @@ Please see the extended architecture documentation [here](/docs/index.md)
 
 k8gb is very well tested with the following environment options
 
-| Type                            | Implementation                                     |
-|---------------------------------|----------------------------------------------------|
-| Kubernetes Version              | >= 1.14 (with install workaround) >= 1.15 (Stable) |
-| Ingress Controller              | Nginx                                              |
-| EdgeDNS                         | Infoblox                                           |
-| Number of k8gb enabled clusters | 2                                                  |
+| Type                             | Implementation                                     |
+|----------------------------------|----------------------------------------------------|
+| Kubernetes Version               | >= 1.14 (with install workaround) >= 1.15 (Stable) |
```

Comment:
Not in the scope, just as you change it, makes sense to provide the reference to install workaround below the table.

Reply:
@somaritane good point, done, please check https://github.com/AbsaOSS/k8gb/pull/186/commits/fa07ac0b86d03c1932f229cc124d11f0c09259f5

Reply:
Looks good!